canonical-web-and-design/ubuntu.com
# Creates VNF page

## Done

- Creates VNF page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco/vnf
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1xq3tFh1vWhNm1gFzddB6LQ65DggVMHsRUSXvqFPqYc0/edit#)
- 'Contact us' form wont work until it's PR is merged


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4190
